shell/common/rasterizer.cc

```diff
@@ -711,22 +711,32 @@ Rasterizer::Screenshot Rasterizer::ScreenshotLastLayerTree(
   }
 
   sk_sp<SkData> data = nullptr;
+  std::string format;
 
   GrDirectContext* surface_context =
       surface_ ? surface_->GetContext() : nullptr;
 
   switch (type) {
     case ScreenshotType::SkiaPicture:
+      format = "ScreenshotType::SkiaPicture";
```

Comment:
I'm not sure how they are used.  I suspect they are strings since the values are read in dart code.  That way we don't have to worry about synchronizing enums values across languages.